models/bow_model.ipynb

```diff
@@ -0,0 +1,564 @@
+     },
+     "execution_count": 104,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "%%time\n",
+    "# Logistic regression time!\n",
+    "logit = LogisticRegression(solver=\"lbfgs\", n_jobs=-1, random_state=7)\n",
+    "logit.fit(X_train, y_train)"
```

Comment:
This call ended early (after 1m 8.2s) with the following error code (not an actual exception, notably):
```
/Users/Will/Documents/ECE 539 Final Project/ECE539-Final-Project/.venv/lib/python3.11/site-packages/sklearn/linear_model/_logistic.py:460: ConvergenceWarning: lbfgs failed to converge (status=1):
STOP: TOTAL NO. of ITERATIONS REACHED LIMIT.

Increase the number of iterations (max_iter) or scale the data as shown in:
    https://scikit-learn.org/stable/modules/preprocessing.html
Please also refer to the documentation for alternative solver options:
    https://scikit-learn.org/stable/modules/linear_model.html#logistic-regression
  n_iter_i = _check_optimize_result(
CPU times: user 1.66 s, sys: 1.9 s, total: 3.56 s
Wall time: 1min 8s
```

Reply:
This is a warning, we can also add more iterations - I have set it to 400 right now